BackEnd/src/db/leaderboards.ts

```diff
@@ -4,7 +4,7 @@ import {GameRoles} from "../../../Common/Interface/General/gameEnums";
 import {FindManyOptions,  MoreThanOrEqual} from "typeorm";
 import {combine} from "../../../Common/utils";
 
-const minAmountOfGames = 4
+const minAmountOfGames = 5
```

Comment:
Any reason for this change? I used 4 because then you would be able to see leaderboards after 2 2-0 series. With this system any 2-0 teams in a league would not be able to see stats until week 3 which is almost half way thru a season.

Reply:
Ah... hm... I didn't think about that. 4 just seemed like a weird number. Will change back.